test/per_file_coverage.sh

```diff
@@ -13,7 +13,7 @@ declare -a KNOWN_LOW_COVERAGE=(
 "source/common/filter/http:96.5"
 "source/common/http/http3:50.0"
 "source/common/json:90.6"
-"source/common/network:95.2"
+"source/common/network:95.0"
```

Comment:
cc @alyssawilk @junr03 

Explanations for this reduction:
- Changes in this PR
- Test coverage for connection_impl.cc went down when runtime flag in https://github.com/envoyproxy/envoy/pull/15028 was rolled back
- IoSocketHandleImpl::activateFileEvents is only covered by a death test, coverage from those is not reliable.
- RawBufferSocket::startSecureTransport not always called by tests. https://github.com/envoyproxy/envoy/pull/13112/files#r578044664

Could you add "# flaky: be careful adjusting" to this line?

Reply:
Added a comment here https://github.com/envoyproxy/envoy/pull/14753/commits/414284dd3e0cbd792817258241c60e764db48bba#diff-0ac04ef31c616358d8234bd473469c1a233ecf37947a13475cce764e7f6c43eeR16